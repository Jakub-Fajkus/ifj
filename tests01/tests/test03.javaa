1200000000000000000000000000000000000000000000
