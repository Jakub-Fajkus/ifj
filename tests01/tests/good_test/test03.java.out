Zadejte cislo pro vypocet faktorialu: Vysledek: 3628800 (zaporny: -3628800)
